README.md

```diff
@@ -15,37 +15,36 @@
+<b>Take a mirror-facing look at bad habits.</b> Jot down your vices, right from the home page. Use notes to add details — like a healthy alternative to that chocolate bar you cannot resist. And organise your input with labels.
 
-<b>Faster than you can grab a pen.</b> Find your bad habits on the home page, and cross out the ones you give into. These will move to Done for the day. The next morning, start off fresh with a full list of things to avoid.
+<b>Faster than you can grab a pen.</b> Find  and cross out the bad habits you give into on the home page. These will move to "Done" for the day. The next morning fresh, with a full list of things to avoid.
 
-<b>Stay on top of your progress.</b> The Log gives you insight into your overall success rates and pinpoints what bad habit you have been most attached to — over a week, a month, or a year. So you can easily work toward a better you.
+<b>Stay on top of your progress.</b> The log grants insight into your overall success rates, pinpointing bad habits you have grown most attached to — over a week, a month, or a year. _Your better you can easily work toward an even better you_.
 
-<a href="https://play.google.com/store/apps/details?id=rocks.poopjournal.todont">
-    <img alt="Get it on Google Play"
-        height="80"
-        src="https://user-images.githubusercontent.com/15004217/36810046-fa306856-1cc9-11e8-808e-6eb8a81783c7.png" />
-        </a>  
 <a href="https://f-droid.org/packages/rocks.poopjournal.todont/">
     <img alt="Get it on F-Droid"
         height="80"
         src="https://user-images.githubusercontent.com/15004217/36919296-19b8524e-1e5d-11e8-8962-48463b1cec8a.png" />
         </a>
+<a href="https://play.google.com/store/apps/details?id=rocks.poopjournal.todont">
+    <img alt="Get it on Google Play"
+        height="80"
+        src="https://user-images.githubusercontent.com/15004217/36810046-fa306856-1cc9-11e8-808e-6eb8a81783c7.png" />
+        </a>
 
 # Contributing
 
-We use the branch ```development``` or a feature branch while developing and push the code for releases to the master/main branch ```trunk```. PRs to the ```trunk``` need at least one approving review before getting merged.
+The ```development``` or a feature branch is used while developing the code, and pushed into the master branch ```trunk``` afterwards for releases.
+PRs to the ```trunk``` need at least one approving review before getting merged.
 
-We use [Crowdin](https://crowdin.com/project/todont) for translations.
+Help translate the app at [Hosted Weblate](https://hosted.weblate.org/engage/todont/).
 
 Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
 
 Please make sure to update tests as appropriate.
 
-Check out the [contribution guidelines](https://github.com/Crazy-Marvin/ToDont/blob/trunk/.github/CONTRIBUTING.md) for details.
```

Comment:
Please do not remove this. The link will work after the next PR into the `trunk` branch.